index.html

```diff
@@ -1795,13 +1774,33 @@ <h2>Resource Integrity</h2>
         </table>
 
         <p>
-Other [[MULTIBASE]] encoding values MAY be used, but interoperability is not
-guaranteed between implementations.
+Other Multibase encoding values MAY be used, but interoperability is not
+guaranteed between implementations using such values.
+        </p>
+      </section>
+
+      <section>
+        <h2>Multihash</h2>
+
+        <p class="issue" title="Multihash may be standardized at IETF">
+The [[?MULTIHASH]] specification has been dispatched at IETF and may be
+standardized. There is active discussion on this initiative in
+the <a href="https://mailarchive.ietf.org/arch/browse/multiformats/">
+Multiformats mailing list at IETF</a>. If the IETF draft is stabilized
+before this specification goes to the Proposed Recommendation phase, the table
+below will be replaced with normative references to the Multihash
+specification. It is the intention of the Working Group to ensure
+alignment between the Multihash values used in this specification and the
+Multihash values defined by the current Multihash community and any potential
+future IETF Multiformats Working Group.
         </p>
 
         <p>
-Common [[MULTIHASH]] header values and their encodings are provided
-below:
+A Multihash value starts with a binary header, which identifies the specific
+cryptographic hash algorithm and parameters used to generate the digest,
```

Comment:
The binary values identifying the particular cryptographic hash algorithms needs to be defined in this specification for people to be able to interoperably implement this.

Reply:
The binary values are defined here:

https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/196.html#multihash